advanced.md

```diff
@@ -45,7 +45,7 @@ Whenever you construct the `UwpFirmata` object yourself, you **must** call `star
 
 ```c#
 //member variables
-IStream bluetooth;
+IStream connection;
```

Comment:
proposing to change just the variable name? My intention is to illustrate that all of our connection types are IStream descendents